testing_and_setup/compass/ocean/scripts/plots/plot_initial_state.py

```diff
@@ -117,7 +125,9 @@ def main():
     plt.text(0, 1, txt, verticalalignment='top', fontproperties=font)
     plt.axis('off')
 
-    plt.savefig(args.output_file_name)
+    plt.tight_layout(pad=4.0)
+
+    plt.savefig(args.output_file_name, bbox_inches='tight', pad_inches=0.1)
```

Comment:
The layout of the figure is a lot better with these settings, taken from MPAS-Analysis climatology map plots.